content/docs/source-git/how-to-source-git.md

```diff
@@ -5,172 +5,364 @@ draft: false
+
+Packit understands a few metadata fields which can be included in the commit
+message, and used to tweak how patch-files are generated and included in the
+spec-file. I include the patch status with the help of the `description`
+field, and specify the patch-file name with `patch_name`.
+
+Amend the last commit...
 
     $ git commit --amend
 
-and append to the end of the commit message
+...and edit the commit message to include this field at the end of it.
 
-```yaml
-present_in_specfile: true
-patch_name: chrony-nm-dispatcher-dhcp.patch
+```
+test/runwrapper: copy the preloaded library
+
+... to a temporary directory because the original location might
+not be accessible by other users.
+
+patch_name: 0001-acl-2.2.53-test-runwrapper.patch
+description: |-
+    avoid permission denied problem with LD_PRELOAD in the test-suite
+
+# Please enter the commit message for your changes. Lines starting
+# with '#' will be ignored, and an empty message aborts the commit.
+#
+# Author:    Kamil Dudka <kdudka@redhat.com>
+# Date:      Tue Jul 3 10:46:58 2018 +0200
+#
+# On branch rawhide
+# Changes to be committed:
+#	modified:   test/runwrapper
+#
 ```
 
-Now packit knows that a patch corresponding to this commit is already applied
-in spec file and its name is "chrony-nm-dispatcher-dhcp.patch".
-
-We can remove the patch file since it's no longer needed:
-
-    $ rm chrony-nm-dispatcher-dhcp.patch
-
-## Building with packit
-
-We almost have all we need to build with packit: we just need to tell packit
-where the spec file is placed in the repository and which git ref represents the
-4.0 release.
-
-    $ cat >.packit.yaml <<EOF
-    ---
-    specfile_path: fedora/chrony.spec
-    upstream_ref: '4.0'
-    patch_generation_ignore_paths: ["fedora/"]
-    EOF
-    $ git add .packit.yaml
-    $ git commit -m "add .packit.yaml"
-
-Let's break down the config file:
-* We tell packit where the spec file is
-* The git tag 4.0 is set as corresponding to the archive defined in the spec
-  file
-* And finally, when packit tries to generate patches for every commit, we
-  indicate that it should ignore changes to the folder "fedora/" (because it
-  would try to create a patch for "chrony.dhclient")
-
-For more info on packit's configuration, please head on to [the configuration page]({{< ref "/docs/configuration.md" >}}).
-
-That's it, the build should now work:
-
-    $ packit srpm
-    Input directory is an upstream repository.
-    Patch chrony-nm-dispatcher-dhcp.patch does not exist
-    100%[=============================>]   195.00   eta 00:00:00 
-    100%[=============================>]     1.63K  eta 00:00:00 
-    SRPM: /home/tt/g/packit/packit.dev/chrony/chrony-4.0-2.g384bb4bd.fc33.src.rpm
-
-now we can rebuild the SRPM in mock
-
-    $ mock --rebuild -r fedora-rawhide-x86_64 chrony-4.0-2.g384bb4bd.fc33.src.rpm
-
+Save and exit.
+
+## Test the source-git repository
+
+You will need Packit version 0.31.0 or higher installed for the following to
```

Comment:
This is not yet released. Should we release it before we reference this doc, or should I rewrite this part in some way?

Reply:
We can probably add some "(which will be released soon)" note and remove it later once @lachmanfrantisek releases it next week ;)

Reply:
I did that and referenced installing the latest development version from Copr.